javacord-api/src/main/java/org/javacord/api/Javacord.java

```diff
@@ -102,6 +102,13 @@
      */
     public static final String DISCORD_GATEWAY_VERSION = "6";
 
+    /**
+     * The voice gateway version from Discord which we are using.
+     * A list with all voice gateway versions can be found
+     * <a href="https://discordapp.com/developers/docs/topics/voice-connections#voice-gateway-versioning">here</a>.
+     */
+    public static final String DISCORD_VOICE_GATEWAY_VERSION = "4";
```

Comment:
```suggestion
    public static final String DISCORD_VOICE_GATEWAY_VERSION = "3";
```

According to the linked documentation version 3 is the latest.
Are you living in the future? 🤔

Reply:
https://github.com/discordapp/discord-api-docs/issues/510
I do